src/main/java/com/commercetools/sync/services/impl/CategoryServiceImpl.java

```diff
@@ -0,0 +1,50 @@
+package com.commercetools.sync.services.impl;
+
+
+import com.commercetools.sync.services.CategoryService;
+import io.sphere.sdk.categories.Category;
+import io.sphere.sdk.categories.CategoryDraft;
+import io.sphere.sdk.categories.commands.CategoryCreateCommand;
+import io.sphere.sdk.categories.commands.CategoryUpdateCommand;
+import io.sphere.sdk.categories.queries.CategoryQuery;
+import io.sphere.sdk.client.BlockingSphereClient;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.models.SphereException;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.List;
+
+public class CategoryServiceImpl implements CategoryService {
```

Comment:
I will do it after this PR is merged. https://github.com/commercetools/commercetools-sync-java/issues/22